src/App.js

```diff
@@ -23,6 +25,8 @@ const App = () => {
         <Route path="/support" component={SupportPage} />
         <ProtectedRoute path="/dashboard" component={Dashboard} />
         <ProtectedRoute path="/settings" component={SettingsPage} />
+        <ProtectedRoute path="/create-plan" component={CreatePlanPage} />
```

Comment:
Rebase on master before merge for this part cos there are updates for line 21

`<Router basename={process.env.PUBLIC_URL} forceRefresh>`